content/hackathon/prd/tranquil-refuge.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Hackathon Project #2: Tranquil Refuge"
+---
+
+## Idea
+
+A place that people that need a physical or sensory reprieve from the environment can go to rest and recharge. Potential to give priority access to people with accessibility needs.

```

Comment:
I think this would sound better as 'A place where people that need....' or 'A place that people who need...'